### ISSUE TITLE
Puput Tags in Other App

### ISSUE THREAD

**Issue by sirrryankennedy** [APSL/puput#49]
I installed puput as a blog app. I also have a home app for my homepage. I'm trying to add a list of recent posts to the html of my homepage, similar to what is found in entries_list.html. 

`{% for entry in 'puput:entries' %}
     <ul class="style3">
	<li><a href="{% pageurl entry %}">{{ entry.title }}</a></li>
     </ul>
{% endfor %}`

This is what I put into my html (I use pageurl because I use slug urls instead of the dates). I've added all of the proper tag loads at the top of homepage.html, I've tried other debugging. Can't seem to figure out why it won't work. I receive the attribute error: 'unicode' object has no attribute 'relative_url'.

**Comment by marctc** [comment]
Are you trying to iterate over entry list with that code?  If you are doing `{% for entry in 'puput:entries' %}` you are iterating over a string. This is way it shows you 'unicode' object has no attribute 'relative_url'. You have to iterate over a queryset or a list of entries.

**Comment by sirrryankennedy** [comment]
I see. I'm just trying to show the latest entries in my homepage.html, similar to what's in entries_list but in my own custom html. I attempted to use just `{% for entry in entries %}` just like in entries_list.html, in addition to `{% for entry in puput:entries %},` and a few other versions. The first shows nothing, the second gives an error. I'm still learning django, I've learned a lot but there's still some gaps in my knowledge. Shouldn't `{% for entry in entries %}` work? I've loaded all the tags at the top. That's what's in entries_list.html. How could I iterate over existing entries in my homepage app?
